### PR TITLE
Research update 2p3p0

### PR DESCRIPTION
**Pull request by zaqqwerty**
Updated tf to 2.3.0 in research branch.  Will wait for #343 to go in so that I can run the notebooks with this update before merging.

### CODE REVIEW

**Comment by MichaelBroughton** [comment]
Can we upgrade to 2.3.1 here ?

File: binary_classifier/binary_classifier.ipynb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "nbformat_minor": 0,
   "metadata": {
     "colab": {
-      "name": "TFQ_Example_BinaryClassifier.ipynb",
+      "name": "Copy of TFQ_Example_BinaryClassifier.ipynb",
```
Comment (MichaelBroughton) [inline binary_classifier/binary_classifier.ipynb:6]: Don't think we want to change the name here.

File: control/control.ipynb [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "nbformat_minor": 0,
   "metadata": {
     "colab": {
-      "name": "TFQ_Example_DNN_LSTM_Qcontrol_application.ipynb",
+      "name": "Copy of TFQ_Example_DNN_LSTM_Qcontrol_application.ipynb",
```
Comment (MichaelBroughton) [inline control/control.ipynb:6]: Copy

**Review by MichaelBroughton — CHANGES_REQUESTED** [review]
Can we change this to 2.3.1 and remove some of the names that have "Copy_***" in them ? After that I think we'll be good to merge this into research.

**Comment by zaqqwerty** [comment]
Denoised this PR in #408

**Pull request closed without merging**